examples/pyplots/sigmoid_function.py

```diff
@@ -0,0 +1,35 @@
+"""
+============
```

Comment:
Sphinx requires the heading markup to have the same length as the heading text. This is what makes the doc build fail:
>/home/circleci/project/doc/gallery/pyplots/sigmoid_function.rst:10: WARNING: Title overline too short.

Reply:
ok, thanks for correcting my beginner mistakes ;)